### ISSUE TITLE
[BUG] `develop-6.x.x` is out of sync with 6.3.0 latest release

### ISSUE THREAD

**Issue by duncdrum** [eXist-db/exist#5549]
According to the [contribution notes](https://github.com/eXist-db/exist/blob/develop/CONTRIBUTING.md#branch-naming) the `develop-6.x.x` is the main branch for ongoing 6.x development. 

`6.3.0` was released from the `6.3.0-branch` but we haven't merged back into `develop-6.x.x` If we want to have a continuous history it should be merge.

**Comment by adamretter** [comment]
> All new features were released with v6.3.0

@line-o No, that is not the case, and at least @dizzzz is aware of that as we discussed it at the time. Please read the Git history.

**Comment by line-o** [comment]
This still needs to be resolved

**Comment by duncdrum** [comment]
@line-o I think we can close this now?
